deliver_project_to_user.sh

```diff
@@ -0,0 +1,77 @@
+#! /bin/bash -l
+
+###
+# This is a simple wrapper around the deliver.py script, which should facilitate the delivery for the SNP platform
+###
+
+# Expects project id, email address and "sensitive" or "not-sensitive" as arguments. Optionally, specify a ";"-delimited list of members email addresses
+PROJECT=$1
+EMAIL=$2
+SENSITIVE="--$3"
+MEMBERS="$4"
+
+# display usage instructions when invoked without arguments
+if [[ -z "$PROJECT" ]]
+then
+  echo ""
+  echo "Deliver SNP genotyping data to SNIC users via GRUS"
+  echo ""
+  echo "    usage: ${0} <PROJECT ID> <PI EMAIL> {sensitive|not-sensitive} ['member_1@email.se;member_2@email.se']"
+  echo ""
+  exit 1
+fi
+
+# Add members
+MEMBER_ARG=""
+if [[ ! -z "$MEMBERS" ]]
+then
+  MEMBER_ARG="--member_email"
+  for a in $(echo "$MEMBERS" |sed -re 's/;/ /g')
+  do
+    MEMBER_ARG="$MEMBER_ARG $a"
+  done
+fi
+
+# set up paths
+GENOTYPEDIR="/proj/ngi2016001/incoming/GENOTYPING"
+STAGINGDIR="$GENOTYPEDIR/staging"
+SUPRCREDS="$GENOTYPEDIR/supr_creds.txt"
```

Comment:
Would it be possible to store the credentials as environment variables instead? Alternatively, encrypt the file, but maybe that's makes it too complicated.

Reply:
it's not ideal but the file is in a safe environment.. I agree that it should be better handled though.. if it's ok with you, I think I'll put this on the todo for next update, also since changing it requires modifying documents in Canea